internal/driver/glfw/key.go

```diff
@@ -0,0 +1,13 @@
+package glfw
+
+// Action represents the change of state of a key or mouse button event
+type Action int
+
+const (
+	// Release Keyboard button was released
+	Release Action = 0
```

Comment:
Note, I know this is internal now so it's not in the public API - but I think it does not need to be exported?